taxonomies/additives.txt

```diff
@@ -23392,27 +23979,40 @@ wikidata:en: Q414089
 
 en: E964, Polyglycitol syrup
 xx: E964
+ar: E964, شراب بولي جليسيتول
+az: E964, Poligliçitol siropu
 bg: E964, Сироп от полиглицитол
+ca: E964, Xarop de poliglicitol
 cs: E964, Polyglycitol sirup
 da: E964, Polyglycitol sirup
 de: E964, Polyglycitol-Sirup
 el: E964, Σιρόπι πολυγλυκιτόλης
 es: E964, Jarabe de poliglicitol
 et: E964, Polüglütsitoolisiirup
-fi: E964, Polyglysitolisiirappi, Polyglysitolisiirappia
```

Comment:
just to make sure, is it intended to remove "Polyglysitolisiirappia"?

Reply:
fixed